mlflow/sagemaker/container/__init__.py

```diff
@@ -12,16 +12,45 @@
 import signal
 from subprocess import check_call, Popen
 import sys
+import yaml
 
 from pkg_resources import resource_filename
 
 import mlflow
 import mlflow.version
 
-from mlflow import pyfunc
+from mlflow import pyfunc, mleap
 from mlflow.models import Model
+from mlflow.utils.logging_utils import eprint
 from mlflow.version import VERSION as MLFLOW_VERSION
 
+MODEL_PATH = "/opt/ml/model"
+
+ENV_KEY_DEPLOYMENT_CONFIG = "deployment_config"
+DEPLOYMENT_CONFIG_KEY_FLAVOR_NAME = "deployment_flavor_name"
+
+DEFAULT_SAGEMAKER_SERVER_PORT = 8080
+
+SUPPORTED_FLAVORS = [
+    pyfunc.FLAVOR_NAME,
+    mleap.FLAVOR_NAME
+]
+
+
+def get_serving_flavor(model):
```

Comment:
I think it's better to handle this outside of the container.

Reply:
From my perspective, it seems like the container should be the source of truth about the flavors it supports. Can you elaborate?

Reply:
I think the more of the logic we can move to the client the better. It simplifies a lot of things. For example, it would be good if you could ask on the client side which flavors you can choose from for a given model to deploy on Sagemaker.